tests/test_cacheutils.py

```diff
@@ -2,7 +2,9 @@
 
 import string
 
-from boltons.cacheutils import LRU, LRI
+import mock
```

Comment:
Tests look pretty good, but the usage of mock seems excessive. We don't need mock functions, a normal function returning a constant should do just fine.